messages/ar.json

```diff
@@ -0,0 +1,38 @@
+{
+	"@metadata": {
+		"authors": [
+			"MusikAnimal"
+			, "Wahrani"
+			, "FShbib"
+			, "Michel Bakni"
```

Comment:
surprisingly this is apparently still valid JSON, but unconventional nonetheless. Should use trailing commas

Reply:
corrected